java/dataflow-load-books/pom.xml

```diff
@@ -57,17 +57,17 @@
     <dependency>
       <groupId>com.google.cloud.bigtable</groupId>
       <artifactId>bigtable-hbase-dataflow</artifactId>
-      <version>1.0.0</version>
+      <version>1.7.0</version>
```

Comment:
This example should eventually replace `bigtable-hbase-dataflow` with `bigtable-hbase-beam`